src/builder.rs

```diff
@@ -53,17 +60,23 @@ pub enum BundleType {
 }
 
 impl BundleType {
-    /// The default bundle type has all flags enabled, and does not require a bundle to be produced
-    /// if no spends or outputs have been added to the bundle.
-    pub const DEFAULT: BundleType = BundleType::Transactional {
-        flags: Flags::ENABLED,
+    /// The default bundle type has all flags enabled, ZSA disabled, and does not require a bundle
+    /// to be produced.
+    pub const DEFAULT_VANILLA: BundleType = BundleType::Transactional {
+        flags: Flags::ENABLED_WITHOUT_ZSA,
```

Comment:
Ditto my previous comment about undoing this to reduce churn:
```suggestion
    pub const DEFAULT: BundleType = BundleType::Transactional {
        flags: Flags::ENABLED,
```